tests/backup/backup_test.go

```diff
@@ -4680,3 +5034,21 @@ func createUsers(numberOfUsers int) []string {
 	wg.Wait()
 	return users
 }
+
+func getEnvString(environmentVariable, defaultValue string) string {
+	value, present := os.LookupEnv(environmentVariable)
+	if !present {
+		value = defaultValue
+	}
+	return value
+}
+
+func getEnvInt(environmentVariable, defaultValue string) int {
```

Comment:
you can write one function and return interface, in the test convert to string or int

Reply:
Done